### PR TITLE
Move errors docs to top nav

### PR DESCRIPTION
**Pull request by tpdorsey**
Moves the Errors docs heading to header navigation as Error Messages.

Also adds Encoded types to the navigation

## Checklist

* [ ] Unit tests for the changes exist
* [ ] Tests pass on CI and coverage remains at 100%
* [x] Documentation reflects the changes where applicable
* [ ] `changes/<pull request or issue id>-<github username>.md` file added describing change
  (see [changes/README.md](https://github.com/samuelcolvin/pydantic/blob/master/changes/README.md) for details)
* [ ] My PR is ready to review, **please add a comment including the phrase "please review" to assign reviewers**

Selected Reviewer: @hramezani

### CODE REVIEW

**Comment by tpdorsey** [comment]
please review

File: mkdocs.yml [hunk 1]
```diff
@@ -83,10 +84,10 @@ nav:
   - usage/dataclasses.md
   - usage/validation_decorator.md
   - usage/postponed_annotations.md
-  - Errors:
-    - Usage Errors: usage/errors.md
-    - Validation Errors: usage/validation_errors.md
   - usage/conversion_table.md
+- Error Messages:
+  - Validation Errors: usage/validation_errors.md
```
Comment (hramezani) [inline mkdocs.yml:89]: Don't we need to have a separate directory for error messages and move these two files there?

Comment (tpdorsey) [reply mkdocs.yml:89]: Navigation is not dependent on the location of the files. 

Does moving them break any existing links from errors to the docs?

Comment (hramezani) [reply mkdocs.yml:89]: > Does moving them break any existing links from errors to the docs?

I don't think but I am not sure

**Review by hramezani — APPROVED** [review]

**Pull request merged**